pyaedt/modeler/Object3d.py

```diff
@@ -223,15 +224,14 @@ def chamfer(self, left_distance=1, right_distance=None, angle=45, chamfer_type=0
 
 
 class VertexPrimitive(EdgeTypePrimitive, object):
-    """Vertex object within the AEDT Desktop Modeler
+    """Contains the vertex object within the AEDT Desktop Modeler.
 
     Parameters
     ----------
-    parent : Object3d
-        Pointer to the calling object which provides additional functionality
-
+    parent : :class:`pyaedt.modeler.Object3d.Object3d`
+        Pointer to the calling object that provides additional functionality.
```

Comment:
This is exactly what all the `parent` parameters should look like.  Note that filling these out is a low priority as these classes are constructed by pyaedt and not the user.